.github/workflows/update_rol_parameters.yml

```diff
@@ -0,0 +1,39 @@
+name: Update ROL Parameters
```

Comment:
Are you sure this should be pushed? It looks like the `find_parameters.sh` script is missing.

Reply:
@gregvw , please advise.

Reply:
@cgcgcg I am confused about your statement that  "`find_parameters.sh` script is missing." It is [here](https://github.com/Rapid-Optimization-Library/ROL-Trilinos/blob/develop/packages/rol/rol_parameters/find_parameters.sh)